.github/workflows/tests_and_qc.yml

```diff
@@ -0,0 +1,57 @@
+# Built from:
+# https://docs.github.com/en/actions/guides/building-and-testing-python
+
+name: Tests and QC
+
+on:
+  workflow_dispatch:
+  # TODO: activate cron when ready
+#  schedule:
+#    - cron: '0 5 * * *'  # every day 5am GMT (12-1am EST/EDT)
+  push:
+    branches: [ develop, main ]
+  pull_request:
+    branches: [ develop, main ]
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+#        os: [ ubuntu-latest, windows-latest ]
+        os: [ ubuntu-latest ]
+#        python-version: [ "3.9", "3.10" ]
+        python-version: [ "3.9" ]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+# todo: do any tests require submodules?
+#        with:
+#          submodules: true
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: 'Create env file'
+        run: |
+          mkdir env
+          echo "${{ secrets.ENV_FILE }}" > env/.env
+      - name: Create and start virtual environment
+        run: |
+          python3 -m venv venv
+          source venv/bin/activate
+# todo: do any tests require submodules?
+#      - name: Git Large File Store
+#        run: |
+#          git lfs install
+#          cd termhub-vocab
+#          git lfs pull
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade wheel
+          pip install --upgrade setuptools
+          pip install -r requirements.txt
+      - name: Run tests
+        run: python -m unittest discover -v
```

Comment:
FYI: The command that runs the tests. I may change this command, as we will likely have 1 action / suite for running "short test" and one for "[long running tests](https://github.com/jhu-bids/TermHub/issues/522)".